src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -429,6 +429,37 @@ def exec_shell(
         """
         return self._true_or_false(entry, config)
 
+    @_post_processor
+    def help_builder(
+        self,
+        entry: SettingsEntry,
+        config: ApplicationConfiguration,
+    ) -> PostProcessorReturn:
+        # pylint: disable=unused-argument
```

Comment:
(accuracy, optional) Is it possible to put this on the same line as the unused argument?

Reply:
It is not required and removed now. Fixed!